jbx-protocol/juice-interface
# Bump NFT contracts

## What does this PR do and why?

Key changes are:
1. `launchProjectFor` args
2. New way of getting a project's tiers (see `useNftRewards` hook)


**How to test:**
1. Launch project with NFTs
2. Check that they load 
3. Pay the project above the amount of an NFT 
4. Check your wallet to see if you received

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist
